src/Whoops/Handler/PlainTextHandler.php

```diff
@@ -284,6 +308,22 @@ private function getTraceOutput()
         return $response;
     }
 
+    /**
+     * Get the exception as plain text.
+     * @param \Throwable $exception
+     * @return string
+     */
+    protected function getExceptionOutput($exception)
```

Comment:
```suggestion
    private function getExceptionOutput($exception)
```

Reply:
if you really insist in private methods.. done